src/actions/receive-profile.js

```diff
@@ -748,20 +759,7 @@ class ColascedSymbolicationStepDispatcher {
   }
 }
 
-const gCoalescedSymbolicationStepDispatcher = new ColascedSymbolicationStepDispatcher();
-
-export function enqueueSingleSymbolicationStep(
-  threadIndex: ThreadIndex,
-  symbolicationStepInfo: SymbolicationStepInfo
-): ThunkAction<void> {
-  return dispatch => {
-    gCoalescedSymbolicationStepDispatcher.enqueueSingleSymbolicationStep(
-      dispatch,
-      threadIndex,
-      symbolicationStepInfo
-    );
-  };
-}
+const _symbolicationStepQueueSingleton = new SymbolicationStepQueue();
```

Comment:
It's amazing how naming can make something much easier to understand, thanks for changing this!